<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_settings_view</class>
 <widget class="QMainWindow" name="user_settings_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Plasmetry - User Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #DDDDDD;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="header">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>74</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>74</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color: #898989;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item alignment="Qt::AlignLeft">
        <widget class="QPushButton" name="back_btn">
         <property name="minimumSize">
          <size>
           <width>104</width>
           <height>58</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>104</width>
           <height>58</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: #FFFFFF;
}</string>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QComboBox" name="probe_selection_cb">
         <property name="minimumSize">
          <size>
           <width>344</width>
           <height>58</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>344</width>
           <height>58</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox {
	background-color: #FFFFFF;
}</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Single Langmuir Probe (SLP)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Double Langmuir Probe (DLP)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Triple Langmuir Probe (TLP)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ion Energy Analyzer (IEA)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hyperbolic Energy Analyzer (HEA)</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>210</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="main_view">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="single_lang_probe">
       <widget class="QLabel" name="debug_label">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>140</y>
          <width>261</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>slp</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="double_lang_probe">
       <widget class="QLabel" name="debug_label_2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>130</y>
          <width>261</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>dlp</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="triple_lang_probe">
       <widget class="QLabel" name="debug_label_3">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>170</y>
          <width>261</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>tlp</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="ion_energy_analyzer">
       <widget class="QLabel" name="debug_label_4">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>140</y>
          <width>261</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>iea</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="hyper_energy_analyzer">
       <widget class="QLabel" name="debug_label_5">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>140</y>
          <width>261</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>hea</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="footer">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>74</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>74</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color: #3C3C3C;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QPushButton" name="reset_btn">
         <property name="minimumSize">
          <size>
           <width>104</width>
           <height>58</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>104</width>
           <height>58</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: #F33A3A;
}</string>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignVCenter">
        <widget class="QLabel" name="alert_msg_label">
         <property name="minimumSize">
          <size>
           <width>560</width>
           <height>72</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>560</width>
           <height>72</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QPushButton" name="save_btn">
         <property name="minimumSize">
          <size>
           <width>104</width>
           <height>58</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>104</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: #FFFFFF;
}</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
